device_calendar/example/lib/main.dart

```diff
@@ -13,10 +10,16 @@ class MyApp extends StatefulWidget {
 class _MyAppState extends State<MyApp> {
   @override
   Widget build(BuildContext context) {
-    return new MaterialApp(routes: {
-      AppRoutes.calendars: (context) {
-        return new CalendarsPage();
-      }
-    });
+    return new MaterialApp(
+      title: 'Device Calendar Example',
+      home: new Scaffold(
```

Comment:
We still need to add some actual examples showing using the calendar in here, right?

Reply:
Yep, I just wanted to check this in, so that we can all agree that this is how the project/example structure should look like.